applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -96,18 +96,25 @@ def _CreateScheme(self):
         scheme_type = self.settings["scheme_type"].GetString()
 
         # Setting the Rayleigh damping parameters
+        rayleigh_alpha = self.settings["rayleigh_alpha"].GetDouble()
+        rayleigh_beta = self.settings["rayleigh_beta"].GetDouble()
+        for prop in self.main_model_part.Properties:
+            prop.SetValue(StructuralMechanicsApplication.RAYLEIGH_ALPHA], rayleigh_alpha)
+            prop.SetValue(StructuralMechanicsApplication.RAYLEIGH_BETA], rayleigh_beta)
+
+        # Setting the Rayleigh damping parameters (legacy on ProcessInfo)
+        KratosMultiphysics.Logger.PrintWarning("ExplicitMechanicalSolver", "RAYLEIGH_ALPHA and RAYLEIGH_BETA are set in explicit simulation, this support is legacy and will be removed at some point")
```

Comment:
Do you agree adding this warning or better just remove it?

Reply:
I'm not an expert on the topic but if I'm not wrong you may compute the damping coefficients based on a previous modal analysis. Then these would be global, that is, the same for all the properties so in my opinion it would make sense to set them through the project parameters.

@RiccardoRossi I'd like to hear your opinion about this.

Reply:
This workflow is already established, see the implicit solver

Reply:
I did changes accordingly to my reasoning